src/i18n/datetimeFormats.ts

```diff
@@ -0,0 +1,52 @@
+import type { DateTimeFormats } from '@intlify/core-base';
+
+const datetimeFormats: DateTimeFormats = {
+  en: {
```

Comment:
maybe you could add a code comment with an example of the format:
// Example: "12/19/2024"